src/expressions.rst

```diff
@@ -4490,20 +4493,8 @@ A :t:`future operand` is an :t:`operand` whose :t:`future` is being awaited by
 an :t:`await expression`.
 
 :dp:`fls_k9pncajmhgk1`
-An :t:`await expression` shall appear within an :t:`async control flow boundary`.
-
-:dp:`fls_xtG3Lw2XgGS2`
-An :dt:`async control flow boundary` is a :t:`control flow boundary` with
-asynchronous control flow.
-
-:dp:`fls_Rr2Wl6mp4oXS`
-The following constructs are :t:`[async control flow boundary]ies`:
-
-* :dp:`fls_xilkqy5piyh0`
-  An :t:`async block expression`.
-
-* :dp:`fls_cr61i8so7cty`
-  The :t:`function body`` of an :t:`async function`.
```

Comment:
We are tossing these out? I don't think we have specified which constructs constitute a `control flow boundary`...

Reply:
They are specified in the corresponding sections, see https://github.com/ferrocene/specification/pull/253/files#diff-302b8fbe696da8535fb26eb2db6211010f45141514fefe8e0e071be8c2ed5b50R100-R102 and https://github.com/ferrocene/specification/pull/253/files#diff-d2497d8980a80566992fa05d12234586c1006c2261a3e39e33f1b50703fc8701R731-R732

Reply:
OK!